addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -36,10 +41,12 @@
  * thing handlers.
  *
  * @author Gaël L'hopital - Initial contribution
+ * @author Ing. Peter Weiss - Welcome camera implementation
  */
 public class NetatmoHandlerFactory extends BaseThingHandlerFactory {
     private Logger logger = LoggerFactory.getLogger(NetatmoHandlerFactory.class);
     private ServiceRegistration<?> discoveryServiceReg;
+    private ServiceRegistration<?> discoveryServiceWelcomeReg;
```

Comment:
There are different api calls for the netatmo devices therefore, and the logic of the api is different, therefore i separated this.